frontend/app/(withCommonLayout)/[...not-found]/page.tsx

```
@@ -0,0 +1,144 @@
+            <div className='error-area'>
+                <Container className='px-4'>
+                    <div className='flex flex-wrap justify-center'>
+                        <div className='w-full lg:w-2/3'>
+                            <div className='text-center md:px-4'>
+                                <div className='error-img'>
+                                    <svg
+                                        viewBox='0 0 649 323'
+                                        fill='none'
+                                        xmlns='http://www.w3.org/2000/svg'
+                                        className='w-full h-full text-theme-secondary'
+                                    >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)